### ISSUE TITLE
No bg-image option

### ISSUE THREAD

**Issue by AnzeBlaBla** [remove-bg/go#2]
Hey! I'm new to github, so sorry if I posted this "issue" in a wrong way or something. I would like to ask you if there's any way to use the bg image option like with curl? If not can you add it? Thanks

**Comment by odlp** [comment]
Hi @AnzeBlaBla. The Go library / CLI doesn't support the background image option yet but I can add this feature for you. Do you intend to specify a background URL or a background image file? (I'll prioritize accordingly).

Thanks,
Oli

**Comment by stefano71** [comment]
Hi, I'm doing something wrong or there is a bug in your app
these are shell commands that I use but unfortunately I get an ever smaller image
"C:\removebg.exe" --api-key myapikey --size auto "C:\001.jpeg"

Thanks,
Stefano

**Comment by odlp** [comment]
@AnzeBlaBla I've just released version `1.2.0` which allows you to provide a background image file:

```
removebg --bg-image-file background.jpg <file>...
```

Alternatively you can can specify a background image URL like this:

```
removebg --extra-api-options "bg_image_url=https://example.com/image.jpg" <file>
```

Please do reach out if you'd like further assistance.
